docs/plugins/verifier/notation.md

```diff
@@ -80,18 +52,53 @@ There are two ways to configure verification certificates:
 1. `verificationCerts`: Notation verifier will load all certificates from path specified in this array.
 
 2. `verificationCertStores`: Defines a collection of Notary Project [Trust Stores](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-store). Notary Project specification defines a [Trust Policy](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md), which is a policy construct to specify which identities and Trust Stores are trusted to produce artifacts in a verification. The name of KeyManagementProvider (KMP) resource(s) must be accurately provided. When a KMP name is specifed, the notation verifier will be configured to trust all certificates fetched from that particular KMP resource. 
-Note: CLI is NOT SUPPORTED.
 
+> NOTE 0: CLI is NOT SUPPORTED.
 
 > NOTE 1: `verificationCertStore` is able to reference a [KeyManagementProvider](../../reference/custom%20resources/key-management-providers.md) to construct trust stores. When referencing a namespaced KMP resource, ensure to include the corresponding namespace prefix, while cluster-wide KMP should be referenced by its name directly. Refer to [this section](../../reference/custom%20resources/key-management-providers.md#utilization-in-verifiers) for more information.
 
-
 > NOTE 2: `verificationCertStores` supersedes `verificationCerts` if both fields are specified.
 
-> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  This change is backward compatible, the implementation supports both original CRs, which contain no trust store type, and CRs specifing `trust-store-type` for each certificate collection. In the sample above, `ca:certs` is the trust store specifing and the `certs` suffix corresponds to the `certs` certificate collection listed in the `verificationCertStores` section. 
+> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  For backward compatibility, users can either specify the truststore type or omit it, in which case it will default to the ca type.
```

Comment:
should `tsa`(coming soon) be replaced wit just tsa?

Reply:
I can remove the coming soon tag after we have our feature branch PR #1758 merged

Reply:
As discussed, this is the doc for the next version, removed (coming soon)